src/runtime/HalideRuntimeHexagonHost.h

```diff
@@ -17,6 +17,8 @@ extern "C" {
  */
 
 #define HALIDE_RUNTIME_HEXAGON
+#define FASTRPC_THREAD_PARAMS (1)
```

Comment:
This (and remote_rpc_thread_params) shouldn't be in this header, they aren't used in the Halide API. I think they should be in hexagon_host.cpp

Reply:
Done. Also I noticed somewhere the default priority value for fastrpc threads was 192 and not 100. I'll check with @pranavb-ca once.